libs/@guardian/source-react-components/vendor/icons/SvgMessage.tsx

```diff
@@ -0,0 +1,43 @@
+// DO NOT EDIT
+// this file is auto-generated by packages/@guardian/source-react-components/scripts/create-icons/index.ts
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { iconSize, visuallyHidden } from '@guardian/source-foundations';
+import type { IconProps } from '../../src/@types/Icons';
+
+const Svg = ({ size }: IconProps): EmotionJSX.Element => (
+	<svg
+		width={size ? iconSize[size] : undefined}
+		height={undefined}
+		viewBox="-3 -3 30 30"
+		xmlns="http://www.w3.org/2000/svg"
+		focusable={false}
+		aria-hidden={true}
+	>
+		<path
+			fillRule="evenodd"
+			clipRule="evenodd"
+			d="M18.75 3v2.571H3V3h15.75Zm-5.464 5.143v2.571H3V8.143h10.286ZM21 15.857v-2.571H3v2.571h18Zm-4.5 2.572v2.57H3v-2.57h13.5Z"
```

Comment:
I am baffled by the lack of concern for the pixel grid in our designs…

Reply:
@mxdvl Let's chat.

Reply:
😁

Reply:
We discussed this offline and this is one of these icons that have been transferred from Illustrator to Figma with low fidelity. Fixing in the authoring software is not easy, hence these unsnapped values.